app/models/periodic_report.rb

```diff
@@ -7,7 +7,7 @@ class PeriodicReport < ApplicationRecord
   # users,
   # nonprofit_id
 
-  belongs_to :nonprofit
+  belongs_to :nonprofit, required: true
```

Comment:
The shoulda-matchers work slightly different in Rails 5. Even a belongs_to is validated as present, the matcher won't work unless I set required. Don't know why.